### PR TITLE
Added thumbnails for icons in conflicts dialog

### PR DESCRIPTION
**Pull request by d2dyno1**
**Resolved / Related Issues**
Items resolved / related issues by this PR.
- Closes #issue...
- Related #issue...

**Details of Changes**
- Added thumbnails for items in conflicts dialog (as per requested @yair100)

**Validation**
How did you test these changes?
- [x] Built and ran the app

**Screenshots (optional)**
image

### CODE REVIEW

File: Files/DataModels/FilesystemItemsOperationDataModel.cs [hunk 1]
```diff
@@ -62,7 +67,15 @@ public List<FilesystemOperationItemViewModel> ToItems(Action updatePrimaryButton
             // Add conflicting items first
             foreach (var item in ConflictingItems)
             {
-                items.Add(new FilesystemOperationItemViewModel(updatePrimaryButtonEnabled, optionGenerateNewName, optionReplaceExisting, optionSkip)
+                var (IconData, OverlayData, IsCustom) = await associatedInstance.FilesystemViewModel.LoadIconOverlayAsync(item.SourcePath, 64u);
```
Comment (gave92) [inline Files/DataModels/FilesystemItemsOperationDataModel.cs:70]: In #5112 I'm removing this method, you can use the identical `FileThumbnailHelper.LoadIconOverlayAsync()`. This way you may not need to pass the `associatedInstance` anymore.

File: Files/DataModels/FilesystemItemsOperationDataModel.cs [hunk 2]
```diff
@@ -62,7 +67,15 @@ public List<FilesystemOperationItemViewModel> ToItems(Action updatePrimaryButton
...
+
+                BitmapImage icon = null;
+                await CoreApplication.MainView.DispatcherQueue.EnqueueAsync(async () =>
```
Comment (gave92) [inline Files/DataModels/FilesystemItemsOperationDataModel.cs:73]: When using `LoadIconOverlayAsync` the returned IconData can be null, better to add a check for this.

Comment (d2dyno1) [reply Files/DataModels/FilesystemItemsOperationDataModel.cs]: Thank you!

**Comment by yaichenbaum** [comment]
@d2dyno1 It looks like the UI isn't aligned properly anymore.

**Comment by d2dyno1** [comment]
> @d2dyno1 It looks like the UI isn't aligned properly anymore.

Hmm.. This already is present on `main`

**Comment by yaichenbaum** [comment]
> > @d2dyno1 It looks like the UI isn't aligned properly anymore.
> 
> Hmm.. This already is present on `main`

This is what it looks like in `main`
image

**Comment by d2dyno1** [comment]
> > > @d2dyno1 It looks like the UI isn't aligned properly anymore.
> > 
> > 
> > Hmm.. This already is present on `main`
> 
> This is what it looks like in `main`
> image

Hmmmmmmm
image

**Comment by yaichenbaum** [comment]
@d2dyno1 I actually can't reproduce this issue at all even with your changes.

**Review by yair100 — APPROVED** [review]

**Comment by yaichenbaum** [comment]
@d2dyno1 Great work! LGTM

**Pull request merged**